LiskHQ/lisk
# Only lint staged files in precommit hook - Closes #1493

### What was the problem?
Commiting ran eslint on files that are not staged

### How did I fix it?
I removed running the `lint` script on precommit, and added `eslint` to the lint-staged scripts

### How to test it?
Changed 2 files, one of them with an error, but don't stage the file with error. Then you can run `node_modules/.bin/lint-staged` to see that it works (`lint-staged` doesn't shout on the invalid file)

### Review checklist
* The PR solves #1493 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
